src/content/docs/opentelemetry/get-started/collector-processing/opentelemetry-collector-processing-intro.mdx

```diff
@@ -13,40 +13,39 @@ redirects:
+The [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) is a vendor-agnostic tool for receiving, processing, and exporting telemetry data. It's recommended to run the Collector in your observability pipeline. While collector requirements and configuration will vary, it's common to route [APM OpenTelemetry data](/docs/opentelemetry/get-started/apm-monitoring/opentelemetry-apm-intro) through a collector. This shifts overhead away from applications, and provides a place to enrich the telemetry with additional environment context data, filter and transform, and more. It's also common to use the Collector for infrastructure monitoring, but this documentation focuses on data processing use cases. See [Collector for infrastructure monitoring](/docs/opentelemetry/get-started/collector-infra-monitoring/opentelemetry-collector-infra-intro) for more information.
 
-Here's an example of the collector as a gateway, but you can also set up the collector as an agent running on the same host as your application:
+Here's an example of the Collector as a gateway, but you can also set up the Collector as an agent running on the same host as your application:
 
 <img
-  width="40%"
   style={{align: "left"}}
   title="Diagram of OTLP with collector"
-  alt="Diagram showing OpenTelemetry using the OpenTelemetry collector and New Relic's OTLP endpoint."
+  alt="Diagram showing OpenTelemetry using the OpenTelemetry Collector and New Relic's OTLP endpoint."
   src="/images/opentelemetry_diagram_native-otlp-with-collector.webp"
 />
 
-The collector has an extensive set of receivers, processors, exporters, extensions, and connectors. These components are bundled into pre-built [distributions](https://opentelemetry.io/docs/collector/distributions/). However, it's possible to build custom components to suit specific requirements, and build [custom collector distribution](https://opentelemetry.io/docs/collector/custom-collector/) to package a specific set of components.
+The collector has an extensive set of receivers, processors, exporters, extensions, and connectors. These components are bundled into pre-built [distributions](https://opentelemetry.io/docs/collector/distributions/). However, it's possible to build custom components to suit specific requirements, and build a [custom collector distribution](https://opentelemetry.io/docs/collector/custom-collector/) to package a specific set of components.
 
-While configuration varys based on requirements, collectors which send data to New Relic should have certain things in common:
+While configuration varys based on requirements, collectors that send data to New Relic should have certain things in common:
```

Comment:
`While configuration varys based on requirements, collectors that send`, the word `varys` should be `varies` and `collectors` is lowercase here.